helper/exceptions/badRequestError.js

```diff
@@ -0,0 +1,25 @@
+class BadRequestError extends Error {
+    constructor(message) {
+        super()
```

Comment:
always pass the argument to your parent class super(message).
Javascript Error has message parameter, you don't need to create a new one in subclass, you can use it.